data/json/items/gun/44.json

```diff
@@ -1,5 +1,34 @@
 [
   {
+    "id": "cane_rifle",
+    "type": "GUN",
+    "name": "walking cane",
```

Comment:
Wouldn't it be better if the name reflected that it's more than just a walking stick?